Task/Task-medication-repeat-reorder.json

```diff
@@ -23,7 +23,7 @@
         ]
     },
     "focus": {
-        "reference": "MedicationRequest/ee86a018-779c-4809-999f-a9d89cdfd30f"
```

Comment:
Do relative links for resource references no longer work or is this a QoL change?

Reply:
This is for workflow elaboration and demonstrating using a shared repository